site/src/components/theme.ts

```diff
@@ -469,16 +470,13 @@ export const theme = createTheme({
         {
           props: { variant: "primary", size: "medium" },
           style: {
-            padding: defaultTheme.spacing(1.25, 2.5),
+            padding: defaultTheme.spacing("12px", "28px"),
           },
         },
         {
           props: { variant: "secondary", size: "medium" },
           style: {
-            paddingTop: `calc(${defaultTheme.spacing(1.25)} - 1px)`,
-            paddingBottom: `calc(${defaultTheme.spacing(1.25)} - 1px)`,
-            paddingLeft: `calc(${defaultTheme.spacing(2.5)} - 1px)`,
-            paddingRight: `calc(${defaultTheme.spacing(2.5)} - 1px)`,
+            padding: defaultTheme.spacing("8px", "24px"),
```

Comment:
Happy for these changes to be merged - but perhaps at some point we can discuss how to best integrate these padding preferences for the remaining button variants and sizes.